src/components/MegadraftEditor.js

```diff
@@ -24,9 +24,11 @@ import DEFAULT_ENTITY_INPUTS from "../entity_inputs/default";
 export default class MegadraftEditor extends Component {
   constructor(props) {
     super(props);
-
     this.state = {
-      readOnly: this.props.readOnly || false
+      readOnly: this.props.readOnly || false,
+      softNewLines: this.props.softNewLines !== undefined
```

Comment:
Hello @Nihaux , thanks for contributing with Megadraft.
I believe you don't need to change the component's state, could you change the code by checking if the prop === false on the handleReturn method?